tests/E2E Tests/AgentApplications/AgentUserIdentityTestscs.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if !FROM_GITHUB_ACTION
+
+using System.Security.Claims;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Graph;
+using Microsoft.Identity.Abstractions;
+using Microsoft.Identity.Web;
+
+namespace AgentApplicationsTests
+{
+    public class AgentUserIdentityTests
```

Comment:
I think this needs to be above the name of the class to take effect
#if !FROM_GITHUB_ACTION